### PR TITLE
:lipstick: automate node setup script and readme update

### PR DESCRIPTION
**Pull request by srinjoychakravarty**
there may be issues that a new user may have trouble with so this is a setup bash script that removes pre-exisiting nodejs installs that may conflict & hard-codes node v10.16.0 that is known to work.

also updates the README to incorporate the setup script (described above) and makes it an easier read.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,59 @@
...
+[![Build Status](https://travis-ci.org/joemccann/dillinger.svg?branch=master)](https://travis-ci.org/joemccann/dillinger)
+
+Hydra is an EOSIO smart contract testing framework that allows you to quickly get started with testing your smart contract code. Besides Hydra, installing additional software or running your own local blockchain node is not required. 
+
+# New Features!
+
+  - Import a HTML file and watch it magically convert to Markdown
+
+You can also:
+  - Import and save files from GitHub, Dropbox, Google Drive and One Drive
```
Comment (cmichel21) [inline README.md:14]: I'm not sure what you're talking about here. Is this Hydra related?

File: README.md [hunk 2]
```diff
@@ -1,15 +1,59 @@
 # hello-hydra
 
-An example eosio.token contract tested with [Hydra](docs.klevoya.com/hydra).
+[![N|Solid](https://cldup.com/dTxpPi9lDf.thumb.png)](https://nodesource.com/products/nsolid)
 
-To install the dependencies run:
+[![Build Status](https://travis-ci.org/joemccann/dillinger.svg?branch=master)](https://travis-ci.org/joemccann/dillinger)
+
+Hydra is an EOSIO smart contract testing framework that allows you to quickly get started with testing your smart contract code. Besides Hydra, installing additional software or running your own local blockchain node is not required. 
+
```
Comment (cmichel21) [inline README.md:8]: Do we need dillinger in this repo? As this is an example repo for new Hydra users, I'd say we should keep the dependencies as lean as possible to not overload the users with additional non-Hydra related information

File: README.md [hunk 3]
```diff
@@ -1,15 +1,59 @@
...
+### Tech
+
+Hydra uses a number of open source projects to work properly:
+
+* [Jest] - a delightful javascript testing framework maintained by facebook with a focus on simplicity
+
+### Installation
+
+Hydra requires [Node.js](https://nodejs.org/) v10.16.0 to run.
```
Comment (cmichel21) [inline README.md:24]: It should work fine with any recent Nodejs version. Also with Node 12. I just set it to Node 10, because Node 8 is not supported anymore.

So I'd say something like this instead:
Hydra requires a recent Node.js version, v10.16.0 or later.

File: README.md [hunk 4]
```diff
@@ -1,15 +1,59 @@
...
+
+Install the dependencies and devDependencies and sign up or login with your existing Github or Google account
```
Comment (cmichel21) [inline README.md:26]: They should also be able to sign up with just an email + password. I 'm not sure if we should mention this here, some people might think GitHub or Google accounts are necessary for Hydra. This might turn off some people

File: README.md (hunk 3 above)
Comment (cmichel21) [reply README.md:24]: If it doesn't work with Node 12 for you, please let me know, we should definitely fix that.

File: README.md [hunk 5]
```diff
@@ -1,15 +1,59 @@
...
+
+```sh
+$ git clone https://github.com/klevoya/hello-hydra.git
+$ cd hello-hydra/
+$ sudo chmod +x setup.sh
+$ sudo bash ./setup.sh
```
Comment (cmichel21) [inline README.md:32]: Imo, the script is a bit too agressive. We shouldn't just uninstall the user's node version and install nvm and use node 10.

Especially, as we don't mention it here and it looks like `setup.sh` is required for any Hydra users. I think a lot of devs already have node installed.

Comment (cmichel21) [reply README.md:32]: The script also only works on Ubuntu / distros with `apt`. Everyone else will see an error when running the script and might think that Hydra doesn't work for them.

I don't think we need the script to be honest. Refering to the official Nodejs installation guide should be enough in my opinion.

File: README.md [hunk 6]
```diff
@@ -1,15 +1,59 @@
 # hello-hydra
 
-An example eosio.token contract tested with [Hydra](docs.klevoya.com/hydra).
+[![N|Solid](https://cldup.com/dTxpPi9lDf.thumb.png)](https://nodesource.com/products/nsolid)
```
Comment (cmichel21) [inline README.md:3]: I think this badge can be removed, we're not using N|Solid

File: README.md [hunk 7]
```diff
@@ -1,15 +1,59 @@
...
+  - Import a HTML file and watch it magically convert to Markdown
+
+You can also:
+  - Import and save files from GitHub, Dropbox, Google Drive and One Drive
+
+### Tech
+
+Hydra uses a number of open source projects to work properly:
+
+* [Jest] - a delightful javascript testing framework maintained by facebook with a focus on simplicity
```
Comment (cmichel21) [inline README.md:20]: I assume you wanted to add a link here for `[Jest]`. It's currently broken

**Comment by MrToph** [comment]
closing this as the node setup is currently not needed in this repo
